crates/oxc_diagnostics/src/service.rs

```diff
@@ -131,7 +133,12 @@ impl DiagnosticService {
         diagnostics: Vec<OxcDiagnostic>,
     ) -> (PathBuf, Vec<Error>) {
         let path = path.as_ref();
-        let source = Arc::new(NamedSource::new(path.to_string_lossy(), source_text.to_owned()));
+        let path_display = path.to_string_lossy();
+        // replace windows \ path separator with posix style one
+        // reflects what eslint is outputting
+        let path_display = path_display.cow_replace('\\', "/");
```

Comment:
When I first started the windows problem I saw the eslint default output prints them in posix style. This will fix all 3-4 tests under windows. 
Would you like to put it into a separat PR to inform the community?

Reply:
This is fine.